Export CALENDAR_SELECTION_TYPE from bpk-component-datepicker

sogame opened the pull request:
When using `bpk-component-datepicker`, it feels weird doing two different imports, specially taking into account `bpk-component-calendar` might not even be used in the project, as it's just an internal dependency that users don't need to know about:

```
import BpkDatepicker from 'bpk-component-datepicker';
import { CALENDAR_SELECTION_TYPE } from 'bpk-component-calendar';
```

I think it would be better to export `CALENDAR_SELECTION_TYPE` in `bpk-component-datepicker`, so only a single export is needed (and only in the package users have installed):

```
import BpkDatepicker, { CALENDAR_SELECTION_TYPE } from 'bpk-component-datepicker';
```

What do you think about this?


Remember to include the following changes:

- [ ] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

olliecurtis commented:
Good spot @sogame, thanks! We can probably do the same for the `bpk-scrollable-calendar` as well!

sogame commented:
> Good spot @sogame, thanks! We can probably do the same for the `bpk-scrollable-calendar` as well!

Added, @olliecurtis :-)
I've also added the import in `readme.md`, as it was missing there 😬
